LICENSE

```diff
@@ -0,0 +1,5 @@
+/* Copyright (C) Dream Faster AI, UG (haftungsbeschränkt) - All Rights Reserved
+ * Unauthorized copying of this file, via any medium is strictly prohibited
+ * Proprietary and confidential
+ * Written by Mark Szulyovszky, Daniel Szemerey, September 2022
+ */
```

Comment:
This is a temporary LICENSE -> ``python3 -m build`` and ``pip install -e .`` (local installation) requires that a ``LICENSE`` is available. This is the most restrictive licence and is actually equal to not having a licence.